File: app/models/snippet.rb
# == Schema Information
#
# Table name: snippets
#
#  id         :integer          not null, primary key
#  inbound    :boolean
#  contact_id :integer
#  context    :text
#  date       :datetime
#  created_at :datetime         not null
#  updated_at :datetime         not null
#  gid        :string
#

class Snippet < ActiveRecord::Base
  belongs_to :user
end

File: app/assets/javascripts/backbone/models/snippet.js
var app = app || {};

app.Snippet = Backbone.Model.extend({
  urlRoot: '/snippets',
  defaults: {
    inbound: 'false',
    contact_id: '0',
    context: '{confidence: "50.0000", result: "Neutral"}',
    date: '2015-04-05T13:11:11.000+10:00',
    gid: '0000000000000000'
  }
});